src/test/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayoutTest.java

```diff
@@ -32,6 +32,9 @@
     private static final String INFO = " INFO  ";
     private static final String ERROR = " ERROR ";
 
+    private static final String LOGGER = String.format("%s:\\d+: ", ReformLoggingLayoutTest.class.getCanonicalName());
+    private static final String LOGGER2 = String.format("%s:\\d+: ", AbstractLoggingException.class.getCanonicalName());
```

Comment:
Those constants only add confusion imho.

Perhaps rename `LOGGER` to `CURRENT_CLASS_NAME` and remove `LOGGER2`?

Reply:
how about `CURRENT_CLASS_LOGGER` and `ERROR_CLASS_LOGGER`

Reply:
is it a 'logger' though?

Reply:
it is in the sense of formatting the log message. you have `%logger` var in logback. all of them are strings in the end

Reply:
`CURRENT_CLASS_LOGGER_NAME` perhaps then?
Either way, I think `":\d+: "` shouldn't be a part of it.

Reply:
It is a part of appended string `log.append(String.format(" %s:%d: ", event.getLoggerName(), lineNumber));`. If there is a need/request to make line numbers behave same as alert level then I can make separate request introducing a flag to show/hide them.

After all, it is a unit test which takes into account unit aka Logger Layout. One should take into account what it stands for. E.g. you see how easily introduced a long variable name right there :). Another pmd rule if I remove the exclusion. Plus giving the unit it is by definition the logger name.

Good point to name it properly since they are not local method variables anymore, but I'm going as much as `CURRENT_CLASS_LOGGER` with line numbers and making separate PR for them if necessary